tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ScalaJSClassEmitter.scala

```diff
@@ -30,7 +31,8 @@ import org.scalajs.core.tools.linker.{LinkedClass, LinkingUnit}
  */
 private[scalajs] final class ScalaJSClassEmitter(
     private[emitter] val outputMode: OutputMode,
-    linkingUnit: LinkingUnit) {
+    linkingUnit: LinkingUnit,
+    private[emitter] val incClassEmitter: IncClassEmitter) {
```

Comment:
I think this is used only in this class, so `private[emitter] val` is unnecessary.